Podfile

```diff
@@ -35,9 +35,9 @@ pod 'Version'
 pod 'XCGLogger'
 
 pod 'Starscream', git: 'https://github.com/zacwest/starscream', branch: 'ha-swift-api'
-pod 'HAKit', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4'
-pod 'HAKit/PromiseKit', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4'
-pod 'HAKit/Mocks', git: 'https://github.com/home-assistant/HAKit.git', tag: '0.4'
+pod 'HAKit', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
+pod 'HAKit/PromiseKit', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
+pod 'HAKit/Mocks', git: 'https://github.com/home-assistant/HAKit.git', branch: 'main'
```

Comment:
Shouldn't this be pinned to a specific commit instead? 

Because with `main` specified, builds at different times might produce different results, which might be unexpected.

Reply:
I'll be updating this as soon as 0.4 is officially released,  currently on pre-release while we are investigating a performance issue